packages/flutter/lib/src/material/scaffold.dart

```diff
@@ -1849,6 +1850,9 @@ class Scaffold extends StatefulWidget {
   ///    Flutter.
   final String? restorationId;
 
+  /// Defines the backgroud box decoration of the [Scaffold].
```

Comment:
This is a little unusual, since now we have Scaffold.backgroundColor as well as Scaffold.backgroundDecoration. The API doc for both properties should mention the other and explain how they interact. The API doc for this property should briefly describe the common use cases for backgroundDecoration.